examples-gallery/example_10_7_betts_test_problems.py

```diff
@@ -0,0 +1,123 @@
+"""
```

Comment:
When you rename a file you need to use `git mv old_filename.py new_filename.py`, that way the old file doesn't remain.

Reply:
The old filename does not show when I do "ls" in git. How can I remove it?
Thanks!

Reply:
I somehow seem to have managed to remove the old file...